js/main.js

```diff
@@ -0,0 +1,219 @@
+
+}
+
+createPage();
+textFadeIn();
+hide();
+GeoSafari();
+
+let zips = []; 
+const APIKey = 'ef757abcb72ab4e6058f4663f531b267';
+const newsCastSound = new Audio('./sounds/News-Sound-Effect.mp3')
+
+
+function saveZips(zipcode) {
+  let zip = {
+    zip: zipcode,
+  }
+  zips.push(zip);
+  console.log(zips)
+  
+  localStorage.setItem('zipcodes', JSON.stringify(zips));
+}
+
+async function getWeatherData(url) {
+  try {
+    const response = await axios.get(url);
+    weatherInfo.loc = response.data.name;
+    weatherInfo.K = Math.round(response.data.main.temp);
+    weatherInfo.F = Math.round(((weatherInfo.K) - 273.15) * (9/5) + 32);
+    weatherInfo.C = Math.round(weatherInfo.K - 273.15);
+    weatherInfo.con = response.data.weather[0].description;
+    weatherInfo.icon = response.data.weather[0].icon;
+    weatherInfo.mCon = response.data.weather[0].main;
+    
+    console.log(weatherInfo.icon);
+
+    loc.innerText = weatherInfo.loc;
+    tempK.innerText = `Kelvin : ${weatherInfo.K}`;
+    tempF.innerText = `Fahrenheit : ${weatherInfo.F}`;
+    tempC.innerText = `Celcius : ${weatherInfo.C}`;
+    condition.innerHTML = `${weatherInfo.con} <img src='./img/${weatherInfo.icon}.png' id="icon">`;
+    
+    textFadeOut();
+    backgroundChange(weatherInfo.mCon);
+    unhide();
+    textFadeIn();
+    newsCastSound.play();
+  } catch (error) {
+    APIerror();
+  }
+}
+
+function backgroundColor(color) {
+  body.style['background-color'] = color;
+}
+
+
+
+function backgroundChange(mainCondition) {
+  console.log(weatherInfo.mCon)
+  void body.offsetWidth;
+  switch (mainCondition) {
+    default:
+      backgroundColor('gray');
+      break;
+    case 'Clear': 
+      backgroundColor('paleturquoise');
+      body.classList.add('fadeIn')
+      break;
+    case 'Clouds': 
+      backgroundColor('lightslategrey');
+      body.classList.add('fadeIn')
+      break;
+    case 'Snow': 
+      backgroundColor('mintcream');
+      body.classList.add('fadeIn')
+      break;
+    case 'Thunderstrom': 
+      backgroundColor('midnightblue');
+      body.classList.add('fadeIn')
+      break;
+    case 'Drizzle': 
+      backgroundColor('steelblue');
+      body.classList.add('fadeIn')
+      break;
+    case 'Rain': 
+      backgroundColor('gray');
+      body.classList.add('fadeIn')
+      break;
+  }
+  setTimeout(body.classList.remove('fadeIn'), 3000)
+}
+
+function hide() {
+  LC.style.visibility = 'hidden';
+  TC.style.visibility = 'hidden';
+  CC.style.visibility = 'hidden';
+  popUp.style.visibility = 'hidden';
+}
+
+function APIerror() {
+  popUp.style.visibility = 'visible';
+  LC.style.visibility = 'hidden';
+  TC.style.visibility = 'hidden';
+  CC.style.visibility = 'hidden';
+}
+
+function popUpRemove() {
+  console.log('go away')
+  popUp.style.visibility = 'hidden';
+}
+
+function unhide() {
+  LC.style.visibility = 'visible';
+  TC.style.visibility = 'visible';
+  CC.style.visibility = 'visible';
+}
+
+function GeoSafari() {
```

Comment:
Be careful with naming. `geoSafari` if you wanna keep this name.

Reply:
name updated